optuna/trial.py

```diff
@@ -148,6 +159,82 @@ def _init_relative_params(self):
             self.study, trial, self.relative_search_space
         )
 
+    @experimental("1.2.0")
+    def suggest_float(self, name, low, high, log=False):
+        # type: (str, float, float, bool) -> float
+        """Wrapper method for ``suggest_uniform`` and ``suggest_loguniform``.
+
+        .. versionadded:: 1.2.0
```

Comment:
The next release version will be `v1.3.0`.
```suggestion
        .. versionadded:: 1.3.0
```